lib/flow/VaporField.cpp

```diff
@@ -124,12 +133,21 @@ VaporField::GetFirstStepVelocityIntersection( glm::vec3& minxyz, glm::vec3& maxx
     const VAPoR::Grid* grid = nullptr;
     std::vector<double> min[3], max[3];
 
+    // For each velocity variables
     for( int i = 0; i < 3; i++ )
     {
-        auto& varname = VelocityNames[i];
-        grid = _getAGrid( 0, varname );
-        VAssert( grid );
-        grid->GetUserExtents( min[i], max[i] );
+        const auto& varname = VelocityNames[i];
+        if( varname.empty() )
+        {
+            min[i].resize( 3, std::numeric_limits<double>::min() );
+            max[i].resize( 3, std::numeric_limits<double>::max() );
+        }
+        else
+        {
+            grid = _getAGrid( 0, varname );
```

Comment:
Here is an instance where if ConstantGrid were an actual grid with dimensions and coordinates this code wouldn't require special handling (conditionals)

Reply:
I changed the behavior of `GetUserExtents()` to return floating point mins and maxes. I guess if we decided to further implement things like `ConstantStructuredGrid`, then it would make sense for those derived classes to keep an actual user extent.